bundles/org.openhab.binding.dali/README.md

```diff
@@ -0,0 +1,63 @@
+# DALI Binding
+
+This binding supports controlling devices on a DALI bus (Digital Addressable Lighting Interface) via a [daliserver](https://github.com/onitake/daliserver) connection.
+
+Daliserver supports the Tridonic/Lunatone DALI USB adapter.
+As it only provides a thin multiplexer for the USB interface, the DALI messages themselves are implemented as part of this binding.
+
+## Supported Things
+
+Currently, these things are supported:
+
+ - daliserver (bridge)
+ - device (single device/ballast on the DALI bus)
+ - group (group of DALI devices)
+ - rgb (virtual device consisting of three directly addressed devices that represent r/g/b (LED) color channels)
+ 
+This binding was tested on a DALI 1-bus with daliserver 0.2.
+
+## Discovery
+
+Automatic device discovery is not yet implemented.
+
+## Thing Configuration
+
+_Describe what is needed to manually configure a thing, either through the (Paper) UI or via a thing-file. This should be mainly about its mandatory and optional configuration parameters. A short example entry for a thing file can help!_
+
+_Note that it is planned to generate some part of this based on the XML files within ```src/main/resources/OH-INF/thing``` of your binding._
+
+### daliserver
+
+| Parameter   | Parameter ID | Required/Optional |  description                           |
+|-------------|--------------|-------------------|----------------------------------------|
+| Hostname    | host         | Required          | IP address or host name of daliserver  |
+| Port Number | port         | Required          | Port of the daliserver TCP interface   |
```

Comment:
I find the double up of information a little confusing at first. Parameter and Paramter ID and I'm guessing it is because the main UI gives a nice label VS textual config gives a camelCase version. I think having both could make someone try and use 'Hostname' instead of `host` in a texual file and then wonder why it does not work.
Usually bindings just give the textual file column as people using the GUI can usually work it out as the labels and descriptions help to guide and clear it up.
I'll leave it up to you what you want to do or if you want to leave as is, just wanted to point out some people speed read and dont fully take the time to understand what is being presented.

Reply:
I think I adopted this table from the skeleton generator -- and since the first column contains whitespace values (i.e. 'Port Number'), I think people will not mistake these as parameter identifiers.